packages/eslint-config-react/index.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
     'react/jsx-props-no-spreading': 0,
     'react/jsx-wrap-multilines': 0,
     'react/prop-types': 0,
+    'react/require-default-props': 0,
```

Comment:
I'm not sure why this started complaining for those two props specifically. I've turned it off as it doesn't really make sense for function components where destructuring is used.